packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -328,7 +339,43 @@ function commitHookEffectListUnmount(tag: number, finishedWork: Fiber) {
   }
 }
 
-function commitHookEffectListMount(tag: number, finishedWork: Fiber) {
+// TODO: Remove this duplication.
+function commitHookEffectListUnmount2(
```

Comment:
Why does GitHub show this function as being newly-added by this branch. It's *in master*. I only added the `nearestMountedAncestor` param.

Reply:
Your branch is not rebased onto master yet. It just contains a commit that has this in it. Maybe you did the rebase in reverse?

Reply:
I merged master into this branch rather than rebase (b'c of the conflicts and I was being lazy). I would have expected that to not show the file changed any more. 🤔 

I just squashed and rebased instead though.